### ISSUE TITLE
Please add a percentage when downloading map tiles

### ISSUE THREAD

**Issue by agx** [rinigus/osmscout-server#406]
Downloading map tiles currently shows numbers:

![image](https://user-images.githubusercontent.com/2052773/182793561-25c872a1-30b3-4c3d-a269-31931f99190b.png)

Having a percentage or progress bar there would be nice to get an idea how far in the process one is.

Thanks for maintaining OSM Scout server!

**Comment by rinigus** [comment]
Agreed, having percentage over there would be nice. I would have to think whether to refactor the code dealing with downloading and keeping map data and having ability to predict how much more storage is needed when you already have some map databases would be handy. That would make getting % for downloads possible as well.

Right now I am busy with the backends (geocoding and visual tiles). After that I could look into this as well.
